src/main/scala-2.12/org/apache/james/gatling/jmap/rfc8621/scenari/PushPlatformValidationScenario.scala

```diff
@@ -0,0 +1,49 @@
+package org.apache.james.gatling.jmap.rfc8621.scenari
```

Comment:
` package org.apache.james.gatling.jmap.scenari`

Because this scenario will mix Draft and RFC-8621...